tapchannel/aux_traffic_shaper.go

```diff
@@ -192,13 +193,19 @@ func (s *AuxTrafficShaper) PaymentBandwidth(htlcBlob,
 	// never be settled. Other HTLCs that may also call into this method are
 	// not yet registered to the commitment, so we need to account for them
 	// manually.
-	computedLocal, _, _, _, err := ComputeView(
+	computedLocal, _, decodedView, _, err := ComputeView(
 		localBalance, remoteBalance, lntypes.Local, htlcView,
 	)
 	if err != nil {
 		return 0, err
 	}
 
+	log.Tracef("Computed asset HTLC View: commitmentLocal=%v, "+
+		"computedLocal=%v, nextHeight=%v, thisHtlc=%v, newView=%v\n",
+		cmsg.OutputSum(commitment.LocalOutputs()), computedLocal,
+		htlcView.NextHeight, htlc.Amounts.Val.Sum(),
+		localViewLogClosure(*decodedView))
```

Comment:
We don't need to pull in `lnutils` here, that was just an example.
Can just be:
```
	log.Tracef("Computed asset HTLC View: commitmentLocal=%v, "+
		"computedLocal=%v, nextHeight=%v, thisHtlc=%v, newView=%v\n",
		cmsg.OutputSum(commitment.LocalOutputs()), computedLocal,
		htlcView.NextHeight, htlc.Amounts.Val.Sum(), func() string {
			return prettyPrintLocalView(*decodedView)
		})
```

Reply:
Also, I don't think we need the `\n` at the end here. If the output needs an extra line at the end, that should come from `prettyPrintLocalView()`.